repos/system_upgrade/el8toel9/models/firewalldusedobjectnames.py

```diff
@@ -0,0 +1,14 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemInfoTopic
+
+
+class FirewalldUsedObjectNames(Model):
+    '''
+    This model contains lists of firewalld object (e.g. zones, services) names
+    in use by the permanent firewalld configuration.
+    '''
+    topic = SystemInfoTopic
+
+    services = fields.List(fields.String(), default=[])
```

Comment:
@erig0 if possible, add please description of the fields (example of some models with proper docs. https://github.com/oamg/leapp-repository/blob/master/repos/system_upgrade/common/models/targetuserspace.py). but it's just nitpick, so keeping it on you.

Reply:
Done